interface.py

```diff
@@ -20,6 +20,12 @@ def __init__(self, dealt_cards, robot_data):
     def __repr__(self):
         return f"InterfaceState Cards: {self.dealt_cards}, My Cards: {self.my_program}, Power Down: {self.power_down,}, Robot: {self.robot_data}"
 
+    def to_server_as_dict(self):
```

Comment:
Zde navrhuji pouze `as dict` jako u ostatních tříd. Tato metoda v sobě neví o serveru, vrací jen slovník. Podle toho by mohl vypadat i docstring.

Reply:
Chtěla jsem to právě rozdělit na dvě casti, druhou jeste nemám. Tyto informace jdou serveru a potom tam bude to_interface _from_dict jestli se nepletu, ale nevim jestli to bude v tom interface ale asi jomto zase bude obsahovat toho robota a karty.

Reply:
tak jsem to promyslela a jo máš pravdu, bude to přehlednější.